ihatemoney/models.py

```diff
@@ -372,6 +373,9 @@ def _to_serialize(self):
             "creation_date": self.creation_date,
             "what": self.what,
         }
+        if self.document:
```

Comment:
nit: not sure why we wouldn't return a "document"/url/... if there is none... like couldn't return a "null"/None?

Reply:
I'm used to this behavior but if you think that it's useful to have a field returning None then I can add it like other fields.